docs/csharp/language-reference/builtin-types/snippets/shared/ValueTuples.cs

```diff
@@ -1,251 +1,270 @@
-﻿using System;
-using System.Collections.Generic;
+﻿// <AliasTupleType>
+global using BandPass = (int Min, int Max);
+// </AliasTupleType>
 
-namespace builtin_types
+namespace builtin_types;
+
+public static class ValueTuples
 {
-    public static class ValueTuples
+    public static void Examples()
     {
-        public static void Examples()
-        {
-            Intro();
-            MethodsOnTuples();
-            LargeTuple();
-            MultipleReturns();
-            ExplicitFieldNames();
-            InferFieldNames();
-            DefaultFieldNames();
-            TupleAssignment();
-            DeconstructExplicit();
-            DeconstructVar();
-            DeconstructExisting();
-            TupleEquality();
-            TupleEvaluationForEquality();
-            TupleAsOutParameter();
-        }
+        Intro();
+        MethodsOnTuples();
+        LargeTuple();
+        MultipleReturns();
+        ExplicitFieldNames();
+        InferFieldNames();
+        DefaultFieldNames();
+        TupleAssignment();
+        DeconstructExplicit();
+        DeconstructVar();
+        DeconstructExisting();
+        TupleEquality();
+        TupleEvaluationForEquality();
+        TupleAsOutParameter();
+        DeconstructToPattern();
 
-        private static void Intro()
-        {
-            // <SnippetIntroduction>
-            (double, int) t1 = (4.5, 3);
-            Console.WriteLine($"Tuple with elements {t1.Item1} and {t1.Item2}.");
-            // Output:
-            // Tuple with elements 4.5 and 3.
-
-            (double Sum, int Count) t2 = (4.5, 3);
-            Console.WriteLine($"Sum of {t2.Count} elements is {t2.Sum}.");
-            // Output:
-            // Sum of 3 elements is 4.5.
-            // </SnippetIntroduction>
-        }
+        // <UseAliasType>
+        BandPass bracket = (40, 100);
+        Console.WriteLine($"The bandpass filter is {bracket.Min} to {bracket.Max}");
+        // </UseAliasType>
+    }
 
-        private static void MethodsOnTuples()
+    private static void DeconstructToPattern()
+    {
+        // <DeconstructToPattern>
+        for(int i = 4; i < 20;  i++)
         {
-            // <SnippetMethodOnTuples>
-            (double, int) t = (4.5, 3);
-            Console.WriteLine(t.ToString());
-            Console.WriteLine($"Hash code of {t} is {t.GetHashCode()}.");
-            // Output:
-            // (4.5, 3)
-            // Hash code of (4.5, 3) is 718460086.
-            // </SnippetMethodOnTuples>
+            if (Math.DivRem(i, 3) is { Remainder: 0, Quotient: var q })
```

Comment:
this uses deconstruction as the article text claims:
```suggestion
            if (Math.DivRem(i, 3) is ( Quotient: var q, Remainder: 0 ))
```